test/utils.jl

```diff
@@ -3,7 +3,11 @@
 mape(x1, x2) = 100 * mean(abs.((x1 .- x2) ./ x1))
 normalized_mape(x1, x2) = mape(x1 / norm(x1), x2 / norm(x2))
 
-function ranking(θ::AbstractVector; rev::Bool=false)
+function argmax_optimizer(θ::AbstractVector; instance::AbstractVector=Float64[])
```

Comment:
Why does the instance have to be an AbstractVector? I would do `instance=nothing` instead

Reply:
I agree